edx_django_utils/monitoring/internal/utils.py

```diff
@@ -143,3 +157,22 @@ def noop_decorator(func):
         return newrelic.agent.background_task(*args, **kwargs)
     else:
         return noop_decorator
+
+
+@contextmanager
+def monitor_django_management_command(name):
```

Comment:
Please document in https://github.com/openedx/edx-django-utils/blob/master/edx_django_utils/monitoring/README.rst.